src/listener/utils/transforms/flowdock.js

```diff
@@ -0,0 +1,142 @@
+/*
```

Comment:
Do you think this eventually could be 'merged' with the HipChat transform to make a single plain-text transform?

Reply:
Yes, I do.